CONTRIBUTORS.md

```diff
@@ -1 +1 @@
-See [TiDB Contributors](https://contributor.tidb.io/people/contributor)
+See [TiDB Contributors](https://ossinsight.io/analyze/pingcap/tidb#contributors)
```

Comment:
Since the contributor info already exists in https://github.com/pingcap/tidb/blob/master/README.md#contributing, I would suggest considering deleting the `CONTRIBUTORS.md` file.